sys/amd64/amd64/pmap.c

```diff
@@ -12311,4 +12312,70 @@ DB_SHOW_COMMAND(ptpages, pmap_ptpages)
 		    (vm_offset_t)pmap->pm_pmltop)), NUP4ML4E, PG_V);
 	}
 }
+
+/***********************
+ * zcond functionality *
+************************/
+struct pmap zcond_pmap;
+vm_offset_t zcond_patch_va;
+
+static void
+pmap_zcond_init(const void *unused) {
+    vm_offset_t kern_start, kern_end;
+    vm_page_t dummy_page;
+
+    kern_start = virtual_avail;
+    kern_end = kernel_vm_end;
+
+	memset(&zcond_pmap, 0, sizeof(zcond_pmap));
+	PMAP_LOCK_INIT(&zcond_pmap);
+	pmap_pinit(&zcond_pmap);
+	pmap_copy(&zcond_pmap, kernel_pmap, kern_start,
+	    kern_end - kern_start, kern_start);
+
+    zcond_patch_va = kva_alloc(PAGE_SIZE);
+    dummy_page = vm_page_alloc_noobj(VM_ALLOC_WIRED);
```

Comment:
```suggestion
    dummy_page = vm_page_alloc_noobj(VM_ALLOC_WIRED | VM_ALLOC_NOFREE);
```

Zasad, pogledat ću možemo li osloboditi stranicu pa updateam komentar.